### PR TITLE
Closes 1823: Remove uppercase text decoration from main nav

### PR DESCRIPTION
**Pull request by cpoff**
Removed uppercase text transform from three nav scss files in AZ Bootstrap.

_navbar.scss
_nav.scss
_navbar-offcanvas.scss

Changed `uppercase` to `none` on all three.

### CODE REVIEW

**Comment by camikazegreen** [comment]
Link to review site: https://review.digital.arizona.edu/arizona-bootstrap/issue/1823-remove-uppercase-nav/docs/2.0/components/navbar/ 

Original docs for comparison: https://digital.arizona.edu/arizona-bootstrap/docs/2.0/components/navbar/

**Review by camikazegreen — APPROVED** [review]
This changes looks good to me.

**Review by mmunro-ltrr — APPROVED** [review]
Checked on the review site.

**Review by danahertzberg — CHANGES_REQUESTED** [review]
Need to run by leadership for approval before merging

**Comment by danahertzberg** [comment]
Add documentation around good navbar use to Bootstrap

**Comment by danahertzberg** [comment]
Change in inspector:
- Non-dropdown menu items

```
.navbar-nav .nav-link {
    font-weight: 600;
    text-transform: none;
    background-color: inherit;
    border: none;
}
```
- Dropdown menu items
```
.navbar-offcanvas .nav-item-parent .nav-link {
    text-transform: none;
}
```

**Review by danahertzberg — APPROVED** [review]

File: scss/_nav.scss
Comment (trackleft) [inline scss/_nav.scss:109]: Should we not just remove the text-transform?

Comment (trackleft) [reply scss/_nav.scss:109]: NVM none is better, since this will add an underline if not there, and adding a text-transform utility class will still work.

**Pull request merged**
